src/main/java/org/kohsuke/github/GHPerson.java

```diff
@@ -52,7 +52,7 @@ protected synchronized void populate() throws IOException {
      */
     public synchronized Map<String,GHRepository> getRepositories() throws IOException {
         Map<String,GHRepository> repositories = new TreeMap<String, GHRepository>();
-        for (GHRepository r : listRepositories()) {
+        for (GHRepository r : listRepositories(100)) {
```

Comment:
Nevermind -- I misread something while running through it.  

Bonus of increasing request size: less likely to hit rate limits, probably.